code/game/objects/items/weapons/dna_injector.dm

```diff
@@ -1190,3 +1190,35 @@
 	New()
 		block = REMOTETALKBLOCK
 		..()
+
+/obj/item/weapon/dnainjector/nofail/randompower/New(newloc)
+    var/type = pick(/obj/item/weapon/dnainjector/nofail/hulkmut,
```

Comment:
It's not that I'm against subtyping the injectors.  I'm against checking every place the injectors are referenced and making sure they still behave as intended.